Documentation/Contributing/CodingGuidelines.md

```diff
@@ -559,6 +559,12 @@ DateTime.UtcNow is faster than DateTime.Now. In previous performance investigati
 
 Prefer using DateTime.UtcNow unless you actually need the localized times (a legitimate reason may be you wanting to show the current time in the user's time zone). If you are dealing with relative times (i.e. the delta between some last update and now), it's best to use DateTime.UtcNow to avoid the overhead of doing timezone conversions.
 
+## PowerShell coding conventions
+
+A subset of the MRTK codebase uses PowerShell for pipeline infrastructure and various scripts and utilities. New PowerShell code should follow the PoshCode style:
```

Comment:
Do we want to remove the raw URL below in favor of an in-line link?

```suggestion
A subset of the MRTK codebase uses PowerShell for pipeline infrastructure and various scripts and utilities. New PowerShell code should follow the [PoshCode style](https://poshcode.gitbooks.io/powershell-practice-and-style/).
```

Reply:
https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/DocumentationGuide.html#links

>Prefer links embedded in a sentence

Reply:
Yes please!